proto/src/main/proto/Rss.proto

```diff
@@ -179,6 +179,14 @@ message ShufflePartitionRange {
   int32 end = 2;
 }
 
+message PMergeContext {
```

Comment:
What's the meaning of `P`? Maybe MergeContext is better.

Reply:
Done